qiskit_nature/components/variational_forms/excitation_op_builder.py

```diff
@@ -0,0 +1,83 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+TODO.
+"""
+
+from typing import Tuple
+
+import itertools
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class ExcitationOpBuilder:
```

Comment:
Maybe this can be an ExcitationBuilder (no Op) and have it be able to build/return simple excitation lists too.

Reply:
Yes, we can decide on the naming at the end of this PR.